website/content/docs/v0.14/Learn More/architecture.md

```diff
@@ -18,6 +18,17 @@ This imposes a clear separation of concerns and ensures that changes over time w
 The benefit is that each component may be iterated and changed as its needs dictate, so long as the external API is controlled.
 This is a key component in reducing coupling and maintaining modularity.
 
+## File system partitions
+
+Talos uses these partions with the following labels:
+
+    - **EFI**: The EFI partition, which is the first partition on the disk and used for EFI boot.
+    - **BIOS**: The BIOS partition, which is the second partition on the disk and used for GRUB's second stage boot.
+    - **BOOT**: The boot partition, which is the third partition on the disk and used for the bootloader.
+    - **META**: The meta partition, which is the fourth partition on the disk and used for storing metadata about the talos node, such as node id's.
+    - **STATE**: The state partition, which is the fifth partition on the disk and used for storing persistent state information.
+    - **EPHEMERAL**: The ephemeral partition, which is the sixth partition on the disk and used for storing ephemeral state information. All the data in this partition can be re-created from the information from the talos machine configuration.
```

Comment:
I think we should clarify it's `/var` partition, and it's not quite true that information can be recreated. 

e.g. etcd database in in the `/var`, and it can recreated only if we have quorum members left. 

container volumes are also going to `/var` by default, so if `/var` is wiped, all container volumes are wiped